configs/setups/awiesm/awiesm.yaml

```diff
@@ -270,6 +292,25 @@ fesom:
                                 "[[wiso_fields-->FIELD]]":
                                         grid: feom
 
+# kh 08.06.22
+                "2.1-recom-par-tracers":
+                        version: "2.1-recom-par-tracers"
+                        ALE_scheme: 'zstar'
+                        #5.7.22 Ying: par-tracers only with ciso
+                        use_ciso: true
+#                        num_tracers: 33
+                        namelist_dir: ${fesom.model_dir}/config/
+                        remove_namelist_changes.namelist.config: [inout]
+                        add_namelist_changes:
+                                namelist.config:
+                                        restart_log:
+                                                restart_length: "${restart_rate}"
+                                                restart_length_unit: "${restart_unit}"
+                        wiso_fields: [o18_feom, hdo_feom, o16_feom, w1_oce, w2_oce, w3_oce, i1_oce, i2_oce, i3_oce] # MA: after merging in FESOM by Martin B. this can be removed
+                        add_coupling_fields:
+                                "[[wiso_fields-->FIELD]]":
+                                        grid: feom
```

Comment:
There is a lot here that is shared with the other version of recom. I suggest we put all the common points in a `choose_with_recom` block and we leave here only the version specific things.